app/app_test.go

```diff
@@ -29,6 +28,8 @@ func TestCurrentApp(t *testing.T) {
 //	assert.Equal(t, "light", fyne.GlobalSettings().Theme())
 //}
 
+// TODO: fix app to avoid summoning the kraken
```

Comment:
This test causes strange GLFW errors for me. I believe that there might be a problem with concurrent starting and stopping. Also, this test used to leave the app running forever.

Reply:
One thought is that we could replace quit with a context passed to Run that would shut the UI down on cancellation.

Reply:
Should this be moved into a separate issue?